F4MPBot/cogs/GithubManager.py

```diff
@@ -0,0 +1,85 @@
+from github import Github
+import discord
+import json
+import logging
+import os
+from discord.ext import commands, menus
+
+COG_DIR = os.path.dirname(os.path.abspath(__file__))
+
+GithubBot = Github("Insert Github Personal Access Token (needs read and write permissions for org)")
+
+
+class GithubCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.log = logging.getLogger(__name__)
+
+    @commands.command(brief='<Project Column Name>',
+                      description='This command is used for looking at F4MP Github project tasks. \n Note: May need '
+                                  'to use quotations for multiword args (i.e "Multi Word Arg")')
+    @commands.has_role('Contributor')
+    async def ProjectTasks(self, ctx, args=""):

```

Comment:
Should use `snake_case` and should also be set as a greedy parameter. `args` as a name is also ambiguous as it implys there are multiple elements within the argument. You should also use `None` as a default value, not an empty string literal.
```suggestion
    async def project_tasks(self, ctx, *, option=None):
    
```